### PR TITLE
created additional help documents

### PR DESCRIPTION
**Pull request by JimmysNetwork**
- Added a 2FA help page explaining how to enable it. 
- Added a Status help page explaining how to better understand our FP status page. 
- Added a Joining Staff help page explaining how to become a staff member for FP. 
- Added a FP Shop help page explaining how to utilize the FP Shop.

### CODE REVIEW

**Review by malcolmjr96 — COMMENTED** [review]
I've made some recommendations on a few things. I still have to review the content, my recommendation is fix the broken link as the top priority so then the tests pass.

File: docs/getting-started/status-page.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: Understanding The Status Page
+permalink: /directory/
+parent: Guide
+---
+
+# How to Read the Forum Promotion Status Page
+
+The Forum Promotion [Status Page](status.forumpromotion.net) provides real-time information about the platform's uptime, any ongoing incidents, and scheduled maintenance. Below is a guide on how to interpret and use the status page effectively.
```
[the remark this answers is not in the document]
Comment (JimmysNetwork) [reply docs/getting-started/status-page.md]: Updated.

File: docs/getting-started/FP-shop.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: FP Shop Guide
+permalink: /directory/
```
[the remark this answers is not in the document]
Comment (JimmysNetwork) [reply docs/getting-started/FP-shop.md]: Updated.

File: docs/getting-started/joining-staff.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Joining Staff Team
+permalink: /directory/
```
[the remark this answers is not in the document]
Comment (JimmysNetwork) [reply docs/getting-started/joining-staff.md]: Updated.

File: docs/getting-started/setup-2fa.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Enabling 2FA
+permalink: /directory/
```
[the remark this answers is not in the document]
Comment (JimmysNetwork) [reply docs/getting-started/setup-2fa.md]: Updated.

**Review by malcolmjr96 — APPROVED** [review]

**Comment by malcolmjr96** [comment]
@JimmysNetwork good to merge 👍

**Pull request merged**